packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -396,30 +396,28 @@ export const routeComponent = (Wrapped, isPage, locals) => {
 }
 
 /**
+ * TODO: update this comment
  * Wrap all the components found in the application's route config with the
  * route-component HOC so that they all support `getProps` methods server-side
  * and client-side in the same way.
  *
  * @private
  */
-export const getRoutes = (locals = {}) => {
-    let _routes = routes
+export const getAllRoutes = async (locals = {}) => {
```

Comment:
I like this name change, it makes things more clear as to what it's doing. Make sure you update the block comment above to further drive it home.